app/scripts/directives/overview/mobileClientRow.js

```diff
@@ -67,6 +67,26 @@
     row.updateServicesInfo = function() {
       if (row.services) {
         row.servicesNotBoundCount = row.services.length - row.bindings.length;
+        var boundServices = 'Bound Services';
+        var unboundServices = 'Unbound Services';
+
+        row.config = {
+          'chartId': 'services-info',
+          'legend': {
+            'show': true,
+            'position': 'right'
+          },
+          colors: {}
+        };
+
+        row.config.colors[boundServices] = '#47b0d6';
```

Comment:
There is a palette available at `$.pfPaletteColors`. Can you check if that has suitable colours?